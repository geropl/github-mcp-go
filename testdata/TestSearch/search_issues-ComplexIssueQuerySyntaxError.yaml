---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 0
        transfer_encoding: []
        trailer: {}
        host: api.github.com
        remote_addr: ""
        request_uri: ""
        body: ""
        form: {}
        headers:
            Accept:
                - application/vnd.github.squirrel-girl-preview
            User-Agent:
                - go-github/v69.2.0
            X-Github-Api-Version:
                - "2022-11-28"
        url: https://api.github.com/search/issues?page=1&per_page=30&q=language%3Ago+AND+bug+is%3Aissue
        method: GET
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: '{"total_count":0,"incomplete_results":false,"items":[]}'
        headers:
            Access-Control-Allow-Origin:
                - '*'
            Access-Control-Expose-Headers:
                - ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
            Cache-Control:
                - no-cache
            Content-Security-Policy:
                - default-src 'none'
            Content-Type:
                - application/json; charset=utf-8
            Date:
                - Wed, 12 Mar 2025 10:54:47 GMT
            Deprecation:
                - Tue, 04 Mar 2025 00:00:00 GMT
            Github-Authentication-Token-Expiration:
                - 2025-04-06 13:14:03 +0200
            Link:
                - <>; rel="deprecation"; type="text/html"
            Referrer-Policy:
                - origin-when-cross-origin, strict-origin-when-cross-origin
            Server:
                - github.com
            Strict-Transport-Security:
                - max-age=31536000; includeSubdomains; preload
            Sunset:
                - Thu, 04 Sep 2025 00:00:00 GMT
            Vary:
                - Accept, Authorization, Cookie, X-GitHub-OTP,Accept-Encoding, Accept, X-Requested-With
            X-Accepted-Github-Permissions:
                - allows_permissionless_access=true
            X-Content-Type-Options:
                - nosniff
            X-Frame-Options:
                - deny
            X-Github-Api-Version-Selected:
                - "2022-11-28"
            X-Github-Media-Type:
                - github.v3; param=squirrel-girl-preview
            X-Github-Request-Id:
                - 57DA:43F75:C3DBDA:C8D474:67D167F7
            X-Ratelimit-Limit:
                - "30"
            X-Ratelimit-Remaining:
                - "23"
            X-Ratelimit-Reset:
                - "1741776900"
            X-Ratelimit-Resource:
                - search
            X-Ratelimit-Used:
                - "7"
            X-Xss-Protection:
                - "0"
        status: 200 OK
        code: 200
        duration: 252.40126ms
