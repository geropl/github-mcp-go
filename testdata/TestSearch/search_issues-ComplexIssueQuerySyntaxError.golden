{
  "output": "# Issue Search Results\n\n**Total Results:** 0\n\nNo issues found.\n",
  "err": ""
}